mesh/meshdb.go

```diff
@@ -137,6 +130,11 @@ func (m *DB) Transactions() database.Getter {
 	return &txFetcher{mdb: m}
 }
 
+// Beacons exports the beacon database.
+func (m *DB) Beacons() database.Store {
```

Comment:
is it a leftover? i think it needs to be removed

Reply:
I've already removed it, but after you started reviewing